src/integrationTest/java/uk/gov/hmcts/darts/workflow/command/DeployRedisCommand.java

```diff
@@ -0,0 +1,49 @@
+package uk.gov.hmcts.darts.workflow.command;
+
+import org.testcontainers.containers.GenericContainer;
+import org.testcontainers.images.builder.ImageFromDockerfile;
+
+import java.io.IOException;
+
+public class DeployRedisCommand implements Command {
+
+    private static GenericContainer<?> container;
+
+    static {
+        ImageFromDockerfile redisImage = new ImageFromDockerfile()
+            .withDockerfileFromBuilder(builder ->
+                                           builder
+                                               .from("redis:7.2.4-alpine")
+                                               .build());
+
+        container = new GenericContainer<>(redisImage);
+        container = container.withExposedPorts(6379);
```

Comment:
Doesn't this need to be a different port number so it doesn't clash with existing redis docker container?

Reply:
This is the internal port number so will not be a problem